MathProblems/LucasNumber.py

```
@@ -0,0 +1,26 @@
+class Solution:
+        def findLucasNumber(self,n:int):
+            def helper(number:int):
+                if number==0:
+                    return 2 
+                if number==1:
+                    return 1 
+                return helper(number-1)+helper(number-2)
+            return helper(n)
+class Solution:
```

Ruff (0.11.9)

Redefinition of unused `Solution` from line 1

(F811)